build.gradle

```diff
@@ -195,6 +207,16 @@ task distributionPackage(type:Zip) {
     }
 }
 
+task robolectricDistributionPackage(type:Zip) {
```

Comment:
perhaps `createRobolectricDistributionPackages` instead? Right now we only have one, but I assume this will be expanded to cover all platforms